internal/controller/replication.storage/volumegroupreplication_controller.go

```diff
@@ -111,6 +111,12 @@ func (r *VolumeGroupReplicationReconciler) Reconcile(ctx context.Context, req ct
 		return reconcile.Result{}, err
 	}
 
+	// Skip reconcile if external VGR
+	if instance.Spec.External {
```

Comment:
I would add a similar check [here](https://github.com/csi-addons/kubernetes-csi-addons/blob/5100f91e506c3c33364d28a644c4e432299318b4/internal/controller/replication.storage/volumegroupreplication_controller.go#L426) such that PVCs that belong to  VGR that is external does not trigger a reconcile for the VGR by this controller.

Reply:
Done